src/app/personside/infotabs/ytelser/YtelserListe.tsx

```diff
@@ -16,22 +18,36 @@ const StyledOl = styled.ol`
     }
 `;
 
+const HjelpetekstHoyreWrapper = styled.div`
+    margin-left: ${pxToRem(50)};
+    position: absolute;
+`;
+
 const TittelWrapper = styled.div`
     padding: ${pxToRem(15)};
     border-bottom: ${theme.border.skille};
+    display: flex;
 `;
 
 interface Props {
     ytelser: Ytelse[];
     pending: boolean;
+    valgtYtelse: Ytelse;
 }
 
 function YtelseListe(props: Props) {
-    const ytelser = props.ytelser.map(ytelse => <YtelserListeElement ytelse={ytelse} />);
+    const ytelser = props.ytelser.map(ytelse => (
+        <YtelserListeElement ytelse={ytelse} erValgt={ytelse === props.valgtYtelse} />
```

Comment:
nice!

Reply:
hvis du åpner consolen din vil jeg tro at du ser at react klager på at dette arrayet med reactnoder mangler keys. Da er det vansklig for react å opptimalisere. Det kan du løse slik:

```
        <YtelserListeElement ytelse={ytelse} erValgt={ytelse === props.valgtYtelse} key={getUnikYtelseKey(ytelse)} />

`
``